### ISSUE TITLE
robots.txt management

### ISSUE THREAD

**Issue by cekk** [plone/volto#2166]
Could be a good idea exposing Plone's robots.txt in Volto instead the empty one in public folder?

In that way we have consistency between frontend and backend and we can update it ttw without rebuilding everything

**Comment by tiberiuichim** [comment]
@cekk: See https://docs.voltocms.com/configuration/environmentvariables/#volto_robotstxt

**Comment by cekk** [comment]
i see, but in Plone we can customize it ttw on control panel: couldn't be nice to have the same behavior in volto without rebuilding the app everytime?

**Comment by tiberiuichim** [comment]
@cekk: you don't have to rebuild the app, it's only read by the server process.

**Comment by ichim-david** [comment]
@cekk I agree with your sentiment and with what @tiberiuichim  mentioned about the volto_robotstxt
I  didn't see your issue before and I've actually left a comment on the recent pull request which made things even less customizable
 https://github.com/plone/volto/pull/2183#issuecomment-778288584

After Volto 11.1.0 robots.txt from public folder takes precedence over volto_robotstxt environment variable and Plone's robots.txt.
The environment variable can be set to something else when starting Volto  and Plone's robots.txt can be edited ttw.

In the absence of having the public folder robots.txt edited it would make sense to change the behavior added in Volto 11.1.0 to something else almost opposite of what is right now.
From :volto public folder -> env variable -> plone robots.txt
To: env variable -> plone robots.txt -> volto public folder
This way you have the most flexibility with what information is displayed since you can set in Plone's robots.txt what you would
have in Volto's robots.txt at any time.

@giuliaghisini how would you feel about such a change proposal?

**Comment by tiberiuichim** [comment]
Ok guys, let's not make this more complicated then this is. The scenario is simple:

- I have a demo site that I want to publish unindexable
- This demo site is stable, but the database is sometimes copied from the production.

The environment variable is the safest option when dealing with dockerized deployments of Volto. It's also something that can be opt-in, you have to specifically set it. So, if you don't care about the above scenario, you go with the second scenario: 

- you have a file robots.txt in public 
- or you don't, in which case Plone's robot.txt should be used.

**Comment by stevepiercy** [comment]
FTR, the docs URL is now https://6.docs.plone.org/volto/configuration/environmentvariables.html#volto-robotstxt
